contentcuration/contentcuration/viewsets/contentnode.py

```diff
@@ -428,11 +429,34 @@ def validate(self, data):
             raise ValidationError(
                 {"parent": "This field should only be changed by a move operation"}
             )
+
+        # Prevent kind from being changed after creation
+        if self.instance is not None and "kind" in data:
```

Comment:
While I do prefer the strict sense of rejecting any request that includes `kind` for an update, it seems less risky to just have this check whether `kind` included in the request is indeed different. I wouldn't expect the inclusion of `kind` in the request, considering how frontend syncing functions, but this strictness could cascade into more required changes if there are surprises.

Reply:
Updated!